src/murmur/ServerDB.cpp

```diff
@@ -725,13 +726,7 @@ void Server::initialize() {
 		query.addBindValue(QLatin1String("SuperUser"));
 		SQLEXEC();
 
-		int length = qrand() % 8 + 8;
-		QString pw;
-		pw.reserve(length);
-
-		while (length--)
-			pw.append(QChar(qrand() % 94 + 33));
-
+		QString pw = PasswordGenerator::generatePassword(12);
```

Comment:
Hmmm. Quite honestly, I think it would hurt readability if we moved the password length too far away from the call site.

I'm OK with making it a constant here though, so it'd be become something like:

    const in passwordLength = 12;
    QString pw = PasswordGenerator::generatePassword(passwordLength);